index.js

```diff
@@ -429,6 +429,42 @@ app.post('/container/:containerId/exec', function(req, res) {
     });
 })
 
+
+/**
+ * Pull the lastest image for the given repository, with or without tag
+ */
+app.get('/pull/:repoTag', function (req, res) {
```

Comment:
We might be better to set this to 

```suggestion
app.get('/pull/*', function (req, res) {
```

From there, we can then determine what to pull. With that above, `req.params[0]` should be set to `homeassistant/home-assistant:rc` for example.

Reply:
👍